pkg/gofr/httpServer.go

```diff
@@ -1,6 +1,7 @@
 package gofr
```

Comment:
Not related to this PR, but I don't think this filename should be in camelCase.

others are using snake case, no?

Reply:
There are a lot of those it a current code base. Let's not fix everything in single PR

Reply:
I created 

- https://github.com/gofr-dev/gofr/issues/794#issue-2392254779